src/mudlet.cpp

```diff
@@ -2698,11 +2698,20 @@ void mudlet::doAutoLogin(const QString& profile_name)
             pHost->mSslTsl = (*it).tlsEnabled;
         }
     } else {
-        QFile file(qsl("%1/%2").arg(folder, entries.at(0)));
+        QFile file(qsl("%1%2").arg(folder, entries.at(0)));
         file.open(QFile::ReadOnly | QFile::Text);
         XMLimport importer(pHost);
         qDebug() << "[LOADING PROFILE]:" << file.fileName();
-        importer.importPackage(&file); // TODO: Missing false return value handler
+        importer.importPackage(&file);
+        if (auto [success, message] = importer.importPackage(&file); !success) {
```

Comment:
Can you not use `pHost->loadedOk` here with a `std::tie(<T1>, <T2>...)`?

Reply:
I can! But I am not sure it would bring much to the table, whereas this is a pattern that's familiar to folks throughout the codebase:

![image](https://user-images.githubusercontent.com/110988/235356597-37a02aff-00b9-4486-82f9-a1dc08d56d7f.png)

Reply:
Fair enough.